isomorphe/static/app.css

```diff
@@ -3,17 +3,50 @@ fieldset {
   border-width: 1px;
 }
 
-.markdown ul + h1,
-.markdown ul + h2,
-.markdown ul + h3,
-.markdown ul + h4,
-.markdown ul + h5,
-.markdown ul + h6 {
-  margin-top: 0.7em;
+.markdown h1,
+.markdown h2,
+.markdown h3,
+.markdown h4,
+.markdown h5,
+.markdown h6 {
+  margin-top: 1em;
 }
 
+
+.markdown p {
+  margin: 0;
+}
+.markdown ul,
+.markdown ol {
+  margin-top: 0.3em;
+  margin-bottom: 0.3em;
+}
+.markdown p + p,
+.markdown ul + p,
+.markdown ol + p {
+  margin-top: 0.6em;
+}
+
+.markdown li {
+  margin-top: 0em;
+  margin-bottom: 0em;
+}
+
+.markdown table {
+  text-align: left;
+  border-collapse: collapse;
+  border-bottom: 1px solid black;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+}
+.markdown table thead tr:last-child {
+  border-bottom: 1px solid black;
+}
+.markdown table tbody tr:not(:last-child) {
+  border-bottom: 1px solid #ddd;
+}
```

Comment:
Did you try with a dsfr table complete markup? https://www.systeme-de-design.gouv.fr/composants-et-modeles/composants/tableau

There's no lightweight option like you've done, but it might be ok. Used elsewhere on the site too.

Reply:
https://github.com/ecolabdata/ecospheres-isomorphe/pull/109/commits/88be04e46da4a9fad1abccdcc99d106d90ef690d :star_struck: